src/main/java/net/dzikoysk/funnyguilds/util/commons/ChatUtils.java

```diff
@@ -1,19 +1,34 @@
 package net.dzikoysk.funnyguilds.util.commons;
 
-import org.bukkit.ChatColor;
+import net.md_5.bungee.api.ChatColor;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.List;
 import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public final class ChatUtils {
+    private static final Pattern HEX_PATTERN = Pattern.compile("#[0-9A-Fa-f]{6}");
 
     private ChatUtils() {}
 
     public static String colored(String message) {
-        return message != null ? ChatColor.translateAlternateColorCodes('&', message) : null;
+        if (message == null) {
+            return null;
+        }
+        
+        Matcher matcher = HEX_PATTERN.matcher(ChatColor.translateAlternateColorCodes('&', message));
+        StringBuilder output  = new StringBuilder();
+
+        while (matcher.find()) {
+            ChatColor color = ChatColor.of(matcher.group());
```

Comment:
Hej! Wydaje mi się, że metoda `ChatColor.of(String)` nie występuje na starszych wersjach minecraft. Dodaj wsparcie dla starszych wersji.

Reply:
Mamy 2021

Reply:
Można dodać sprawdzanie wersji serwera i po tym kolorować z hex lub nie.